internal/cli/atlas/clusters/upgrade.go

```diff
@@ -113,8 +113,8 @@ func UpgradeBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "upgrade [clusterName]",
 		Short: "Upgrade a MongoDB Atlas cluster's tier, disk size, and/or MongoDB version.",
```

Comment:
since you are touching this file we recently got a support to clarify this command is only available for shared tier clusters? if not we can do as a follo wup